# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsndfile/autoconf-fix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libsndfile-1.0.17-orig/configure.ac libsndfile-1.0.17/configure.ac
--- libsndfile-1.0.17-orig/configure.ac	2008-04-19 13:55:59.000000000 +0000
+++ libsndfile-1.0.17/configure.ac	2008-04-19 13:56:46.000000000 +0000
@@ -30,6 +30,7 @@
 
 SHARED_VERSION_INFO="1:17:0"
 
+AC_PROG_CXX
 AC_PROG_CC
 AM_PROG_LIBTOOL
 
